src/elements/helpers/html.ts

```diff
@@ -13,3 +13,12 @@ export const parseHtml = (html: string) => {
   );
   return parsedHtml.body.firstElementChild;
 };
+
+export const isHtmlSafe = (html: string) => {
```

Comment:
One thing – might be nice to add a test for this, if only to clarify what it's for!

Reply:
(And possibly rename to make it clearer what this does – `htmlContainsSingleIframe` ? 😅 )

Reply:
Good suggestion, will do so!